pkgs/tools/misc/bkt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ rustPlatform
+, fetchFromGitHub
+, lib
+, doCheck ? true
+, withFeatures ? p: p
```

Comment:
In case there will be a default set of features, this can be used to override or extent this default set; if we have a list we can only replace *or* append.

Reply:
Can't you just use `overrideAttrs` like on all other packages? I'm also not entirely sure how features interact with cargoHash, surely if features pull in more dependencies this hash would also need updating anyways?

Reply:
> surely if features pull in more dependencies this hash would also need updating anyways

yes, I think this is the case

> Can't you just use overrideAttrs like on all other packages

I'm not sure; the issue is that `overrideAttrs` overrides the set that is passed to `mkDerivation` and not to `buildRustPackage`, I had some issues with this in the past.

Reply:
Hi, any news here? Should I change this?